fog/ingest/server/src/controller.rs

```diff
@@ -1198,14 +1199,20 @@ where
         state: &mut MutexGuard<IngestControllerState>,
     ) -> Result<IngressPublicKeyStatus, Error> {
         // Get a report and check that it makes sense with what we think is happening
-        let report = {
-            let report = self.enclave.get_attestation_evidence()?;
+        let evidence = {
```

Comment:
Same as other comment: out-of-scope for this PR

Reply:
I guess this would get cleaner if the enclave API returns the specific variant it supports instead of the enum (where we know the other variant will never be returned).